service/src/main/resources/db/migration/V371__partner_modifier_column.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TYPE create_type AS ENUM ('USER', 'APPLICATION');
+CREATE TYPE modify_type AS ENUM ('USER', 'DVV');
+
+ALTER TABLE fridge_partner
+    ADD COLUMN create_type create_type default null,
+    ADD COLUMN created_at timestamp with time zone default null,
```

Comment:
It's better to rename the existing `created` column to `create_at` and drop the default value.

Reply:
I think we should also consider renaming `updated` -> `modified_at` or copying values from updated to modified_at.